opendut-util/src/logging.rs

```diff
@@ -46,16 +55,85 @@ pub fn initialize_with_config(config: LoggingConfig) -> Result<(), Error> {
             None
         };
 
+
+    let opentelemetry_layer = config.opentelemetry_endpoint.clone().map(|endpoint| {
+        let otlp_exporter = opentelemetry_otlp::new_exporter()
+            .tonic().with_endpoint(endpoint.url);
+        let tracer = opentelemetry_otlp::new_pipeline()
+            .tracing()
+            .with_exporter(otlp_exporter)
+            .with_trace_config(
+                opentelemetry_sdk::trace::config().with_resource(Resource::new(vec![KeyValue::new(
+                    opentelemetry_semantic_conventions::resource::SERVICE_NAME,
+                    "OTel",
```

Comment:
Introduce config parameter for the `SERVICE_NAME` value.

Reply:
The parameter for SERVICE_NAME was moved to carl(-development).toml and set to "OTel" there